jekyll/_cci2/google-auth.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: classic-docs
+title: Authenticating Google Cloud Platform
+description: Authenticating Google Cloud Platform
+---
+
+Before you can use the `gcloud` command line tool with CircleCI, you must authenticate it. To do this, you will need to create a [service account][]. You can then add this account as an [environment variable]({{ site.baseurl }}/2.0/env-vars/) within CircleCI. Your build script will decode this variable and authenticate the `gcloud` tool for use in your project.
+
+## Prerequisites
+
+- A Google account.
+- A Google Cloud Platform project.
+
+## Steps
+
```

Comment:
Add a summary sentence of the result of completing these steps after the heading.

Reply:
Isn't that what the overview is for?